manifest.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
What's this file for?

Reply:
It is a deployment descriptor for CF. Take a look here for more information: https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html

Reply:
Ok, but why does it need to be included in the repo?

Reply:
It defines basic necessities for the deployment, like the CF buildpack and the command (with flags) to be executed.
In the end it makes the deployment easier, as relevant stuff is predefined.

Reply:
Ok. Can you move the file intro contrib and leave a note in the README?

Reply:
Ok